test/conformance/virtual_memory/urVirtualMemReserve.cpp

```diff
@@ -2,3 +2,61 @@
 // Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <uur/fixtures.h>
+
+using urVirtualMemReserveTestWithParam =
+    uur::urVirtualMemGranularityTestWithParam<size_t>;
+UUR_TEST_SUITE_P(urVirtualMemReserveTestWithParam,
+                 ::testing::Values(2, 4, 8, 16, 32, 64, 128, 256, 512, 1024,
+                                   2048, 5000, 100000),
+                 uur::deviceTestWithParamPrinter<size_t>);
+
+TEST_P(urVirtualMemReserveTestWithParam, SuccessNoStartPointer) {
+    // round up to nearest granularity
+    size_t virtual_mem_size =
```

Comment:
Maybe I'm missing something but I don't see granularity being set anywhere in the code. If it is 0, then RoundUpToNearestFactor() is going to do a division by 0 which is undefined behaviour.

Also, why do we need this RoundUpToNearestFactor() function if the granularity never changes?

Reply:
I see. I missed that call in the fixture. Looks good then :)